src/components/tags/tag-item.html

```diff
@@ -1,5 +1,5 @@
 <a class="tags__list-item drawer-menu__list-item mdc-list-item" href="#" data-name="<%= tag %>">
   <i class="material-icons mdc-list-item__graphic" aria-hidden="true">local_offer</i>
-  <%= tag %>
+  <p class="tags__list-item-name"><%= tag %></p>
```

Comment:
remove `<p class="tags__list-item-name"><%= tag %></p>` and leave only `<%= tag %>`